src/DAL/migration/storedProcedure.sql

```diff
@@ -22,54 +22,11 @@ BEGIN
     IF NOT is_valid THEN
         RAISE EXCEPTION 'Invalid geometry extent: %', ST_Extent(r.geom);
     END IF;
-
-    -- create (if not exists) a temp table to hold results instersection result for the current input
-    CREATE TEMPORARY TABLE IF NOT EXISTS selected_parts ON COMMIT DROP AS (
-        SELECT ST_IsEmpty(geom_intersections) is_empty_geom, ST_NumGeometries(geom_intersections) > 1 is_multi, *
-        FROM (
-            SELECT "internalId", ST_Difference(geom, r.geom) geom_intersections
-            FROM "PolygonParts".parts 
-            WHERE ST_Intersects(geom, r.geom)
-        ) q
-    ) WITH NO DATA;
-
-    -- insert intersecting polygon parts into a temp table
-    INSERT INTO selected_parts
-	SELECT ST_IsEmpty(geom_intersections) is_empty_geom, ST_NumGeometries(geom_intersections) > 1 is_multi, *
-    FROM (
-        SELECT "internalId", ST_Difference(geom, r.geom) geom_intersections
-        FROM "PolygonParts".parts 
-        WHERE ST_Intersects(geom, r.geom)
-    ) q;
-
-    -- insert multi polygons that are the result of intersection with the input polygon
-    INSERT INTO "PolygonParts".parts(geom, "recordId", "productId", "productName", "productVersion", "sourceDateStart", "sourceDateEnd", "minResolutionDeg", "maxResolutionDeg", "minResolutionMeter", "maxResolutionMeter", "minHorizontalAccuracyCE90", "maxHorizontalAccuracyCE90", sensors, region, classification, description, "imageName", "productType", "srsName")
-    SELECT "parts_geom" geom, "recordId", "productId", "productName", "productVersion", "sourceDateStart", "sourceDateEnd", "minResolutionDeg", "maxResolutionDeg", "minResolutionMeter", "maxResolutionMeter", "minHorizontalAccuracyCE90", "maxHorizontalAccuracyCE90", sensors, region, classification, description, "imageName", "productType", "srsName"
-    FROM (SELECT (ST_Dump(geom_intersections)).geom parts_geom, p.*
-        FROM selected_parts
-        JOIN "PolygonParts".parts p
-        ON selected_parts."internalId" = p."internalId" AND selected_parts.is_multi IS true) q;
-
-    -- update geometries of polygon parts intersecting with input polygon
-    UPDATE "PolygonParts".parts
-    SET geom = selected_parts.geom_intersections
-    FROM selected_parts
-    WHERE parts."internalId" = selected_parts."internalId" AND selected_parts.is_empty_geom IS false AND selected_parts.is_multi IS false;
     
-    -- delete completely covered polygon parts by the input polygon
-    DELETE FROM "PolygonParts".parts
-    WHERE (parts."internalId", true) IN (SELECT "internalId", is_empty_geom FROM selected_parts) OR
-    (parts."internalId", true) IN (SELECT "internalId", is_multi FROM selected_parts);
-    
-    -- insert the imput record
```

Comment:
All of this was replaced with the `plsql` in the `functions` file, right?

Reply:
yes, but now it's irrelevant